_docs/geyser/faq.md

```diff
@@ -19,7 +19,7 @@ No, Geyser and all related projects are completely free and open-source, license
 
 Geyser should generally work fine with plugins, as we emulate a Java client. There are exceptions, though:
 
-* [TCPShield](https://tcpshield.com/) requires `only-allow-proxy-connections` disabled without a paid plan. However, their "Premium" plan will allow the support of Geyser - please contact their support for help setting this up.
+* [TCPShield](https://tcpshield.com/) requires `only-allow-proxy-connections` disabled without a paid plan. However, their "Premium" plan will allow the support of Geyser - please contact their support for help setting this up. An alternative that works with Geyser is [Cosmic Guard](https://cosmicguard.com/)
```

Comment:
In context, this sounds a lot like CosmiGuard is a free solution, which it isn't. Also, why should TCPshield point to CosmicGuard, but not the CosmicGuard to TCPshield? I'd remove this note entirely tbh.

Reply:
Sure thing